packages/front-end/components/Features/FeatureModal/FeatureFromExperimentModal.tsx

```diff
@@ -172,7 +174,11 @@ export default function FeatureFromExperimentModal({
   let ctaEnabled = true;
   let disabledMessage: string | undefined;
 
-  if (!permissions.check("createFeatureDrafts", project)) {
+  if (
+    !permissionsUtil.canManageFeatureDrafts({
+      project: experiment.project || project,
```

Comment:
This fixes a bug.

When duplicating a feature, we were checking the current active project (via the drop-down in the left nav). Now, if the user is trying to duplicate a feature, we check the project the feature is in.

This was a bug if the user had a global `noaccess` role, and multiple project-specific roles - if they had `All Projects` selected in the left-nav, it was then checking the user's global role for permission, when the feature was trying to be created in the experiment's project.

Reply:
Same here `?? project` instead of `|| project`